docs/SpatialEpigenome.rst

```diff
@@ -279,19 +269,14 @@ in specific regions of the tissue, visualize it's spatial distribution, and gain
 ------------------------------------------------------------------------
 
 **Add group coverages**
-Use the addGroupCoverages function to call on the input ArchR project object, proj_in_tissue, to add group coverages to the object. The groupBy 
-parameter specifies which metadata column to group the coverages by, in this case 'Clusters'.  
-This function calculates the average coverage of each peak in a single-cell RNA-seq dataset, grouped by 'Clusters'. The resulting object contains the 
-average coverage of each peak in each group, along with metadata about the peaks and the groups. This is done to obtain an understanding of the 
-coverage of peaks across different clusters to provide insights into various cluster characteristics.::
+
+It's important to understand how gene expression varies across different cell types or clusters. The addGroupCoverages function calculates the average coverage of each genomic region (or "peak") in the dataset, grouped by a specific metadata column (in this case, Clusters). By doing so, it allows us to compare the average expression of each peak across different cell types, providing insights into various cluster characteristics.::
 
     proj_in_tissue <- addGroupCoverages(ArchRProj = proj_in_tissue, groupBy = "Clusters")
 
 **Call peaks using MACS2** 
-Use the findMacs2 function to find the path to the macs2 program on the system. Then, use the addReproduciblePeakSet function to call on the 
-proj_in_tissue to add reproducible peak sets to the object. The groupBy, pathToMacs2, and genomeSize parameters are used to specify the metadata column 
-to group the peaks by, the path to the macs2 program, and the size of the genome. The force parameter is set to TRUE to force re-running the peak 
-calling even if it has already been performed. This step is done to identify reproducible peaks across different clusters.::
+
+It's also important to identify genomic regions that are consistently expressed across multiple samples. These are called "peaks" and can be identified using software like MACS2. The addReproduciblePeakSet function to call peaks on our dataset (proj_in_tissue) and add the results to the object. To use this function, we pass in the ArchRProj object, the name of the metadata column to group the peaks by (Clusters), the path to the MACS2 program, and the size of the genome we're analyzing. We also set the force parameter to TRUE, which tells the function to re-run peak calling even if it's already been done (useful if we've made changes to our dataset).::
```

Comment:
Better to specify **accessible** genomic regions. Also, change "The addReproduciblepeakSet function to ..." to  "We call the addReproduciblePeakSet function to ..."